src/Bookings.js

```diff
@@ -1,18 +1,40 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+
+const Bookings = ({ newBooking }) => {
+  const [bookings, setBookings] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    fetch(`https://cyf-react.glitch.me`)
+      .then(res => res.json())
+      .then(data => {
+        setBookings(data);
+        setLoading(true);
+      })
+      .catch(error => window.alert("Ooops, Something went wrong"));
+  }, []);
 
-const Bookings = () => {
   const search = searchVal => {
-    console.info("TO DO!", searchVal);
+    let searchRes = bookings.filter(booking => {
```

Comment:
This looks good to me too , well done 👍 . Just remember that the update on the UI isn't happening because you're saving props in state in one of the child components.